### PR TITLE
Student-teacher list

### PR DESCRIPTION
**Pull request by tonkaew131**
#41 #18

### CODE REVIEW

**Review by Gapraomoosap — APPROVED** [review]
image

File: learn-arai/components/hooks/useClassroom.ts [hunk 1]
```diff
@@ -234,6 +234,19 @@ export const useClassroom = () => {
...
+    const getUsers = async (slug: string) => {
+        const response = await fetch(
+            `${process.env.NEXT_PUBLIC_BACKEND_URL}/c/${slug}/members`,
+            {
+                method: 'GET',
+                credentials: 'include',
+            }
+        );
+        const data = await response.json();
+        console.log(data);
```
Comment (MonitorIizard) [inline learn-arai/components/hooks/useClassroom.ts:246]: Should we removed `console.log`.

**Comment by tonkaew131** [comment]
> image

Lmao literally Peerasin

File: learn-arai/components/hooks/useClassroom.ts (hunk 1 above)
Comment (tonkaew131) [reply learn-arai/components/hooks/useClassroom.ts]: Removed

**Review by MonitorIizard — APPROVED** [review]

**Pull request merged**